tests/run-make/extern-fn-reachable/rmake.rs

```diff
@@ -0,0 +1,55 @@
+//! Smoke test to check that that symbols of `extern "C"` functions and `#[no_mangle]` rust
+//! functions:
+//!
+//! 1. Are externally visible in the dylib produced.
+//! 2. That the symbol visibility is orthogonal to the Rust nameres visibility of the functions
+//!    involved.
+
+//@ ignore-cross-compile
+
+use run_make_support::object::{self, Object, ObjectSection, ObjectSymbol};
+use run_make_support::{dynamic_lib_name, is_darwin, is_msvc, path, rfs, rustc};
+
+fn main() {
+    let dylib = dynamic_lib_name("dylib");
+    rustc().input("dylib.rs").output(&dylib).arg("-Cprefer-dynamic").run();
+    let mut fun1_count = 0; // #[no_mangle] pub extern "C" fn fun1() {}
+    let mut fun2_count = 0; // #[no_mangle] extern "C" fn fun2() {}
+    let mut fun3_count = 0; // mod foo { #[no_mangle] pub extern "C" fn fun3() {} }
+    let mut fun4_count = 0; // pub mod bar { #[no_mangle] pub extern "C" fn fun4() {} }
+    let mut fun5_count = 0; // #[no_mangle] pub fn fun5() {}
+    let mut fun6_count = 0; // #[no_mangle] fn fun6() {}
+
+    let p = if is_msvc() { path(dylib) } else { path(dylib) };
```

Comment:
is this a typo or expected?

Reply:
Oops, this was a leftover from trial and error

Reply:
Removed.